src/App.tsx

```diff
@@ -27,6 +28,22 @@ const HomeStack = () => {
       <Stack.Screen name="WelcomeScreen" component={WelcomeScreen} />
       <Stack.Screen name="LoginScreen" component={LoginScreen} />
       <Stack.Screen name="SignUpScreen" component={SignUpScreen} />
+      <Stack.Screen
+        name="PasswordChanged"
+        component={PasswordChanged}
+        options={({ navigation }) => ({
+          headerTitle: '',
+          headerShadowVisible: false,
+          headerLeft: () => (
+            <TouchableOpacity onPress={() => navigation.goBack()} style={{ marginLeft: 15 }}>
```

Comment:
Same feedback I left for Shilpa :) https://github.com/LittleTechBookClub/swarm-ios-app/pull/25#discussion_r1628746639